src/models/record.py

```diff
@@ -88,6 +90,65 @@ def find_phone(self, phone_number: str) -> Phone:
             if phone.value == phone_number:
                 return phone
         return None
+    
+    def add_note(self, note: str, name: str = None) -> None:
+        """
+        Adds a new note to the record with an optional name.
+
+        Args:
+            note (str): The content of the note to add.
+            name (str, optional): The name associated with the note.
+        """
+        self.notes.append(Note(note, name))
+
+    def edit_note(self, old_name: str, new_note: str) -> None:
```

Comment:
def edit_note(self, old_name: str, new_note: str)

Args:
            old_name (str): The name of the note to be replaced.
            new_note (str): The new content to replace the old note.
            new_name (str, optional): The new name associated with the note.
            
List of parameters does not match descripted comment parameters list